### PR TITLE
Description for OONI's mobile app

### PR DESCRIPTION
**Pull request by agrabeli**
This includes a description for OONI's mobile app (Android & iOS). It also includes an extra short description for the Android version, and some keywords. The description will be displayed on the app when it's released.

### CODE REVIEW

File: test-descriptions/ooni-mobile-app.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# OONI mobile
+
+The OONI mobile version for Android and iOS can be called **OONI-probe: Test
```
Comment (hellais) [inline test-descriptions/ooni-mobile-app.md]: The correct capitalisation here is `ooniprobe`

File: test-descriptions/ooni-mobile-app.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+world.
+
+* **Collecting evidence of Internet censorship.** OONI's web connectivity test is
+designed to examine whether websites are blocked and if so, how. This test, in
+particular, attempts to determine whether access to sites is blocked through DNS
+tampering, TCP connection RST/IP blocking, or by a transparent HTTP proxy. By
+knowing how access to sites is interfered with, you can more easily evaluate how
+to circumvent that specific type of censorship. As OONI is committed to
+transparency through the publication of all network measurement data, you can
+use it as evidence of any censorship events that you come across.
```
Comment (hellais) [inline test-descriptions/ooni-mobile-app.md]: This sentence sounds a bit funny to me.

File: test-descriptions/ooni-mobile-app.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+surveillance, and traffic manipulation. OONI's HTTP invalid request line test is
+designed to uncover the presence of such systems within tested networks.
+However, it's important to point out that not all the systems that you might
+find are necessarily responsible for censorship and/or surveillance. Many proxy
+technologies, for example, are used in networks for caching purposes.
+
+* **Measuring the speed and performance of your network.** Sometimes the network that
+we are using doesn't work as well as we'd like it to. OONI's implementation of
+the Network Diagnostic Test (NDT) attempts to measure the speed of your network
+by connecting to mLab servers near you and by subsequently uploading and
```
Comment (hellais) [inline test-descriptions/ooni-mobile-app.md]: I think the correct spelling for mLab is "M-Lab".

Comment (hellais) [inline test-descriptions/ooni-mobile-app.md]: (possibly add link as well)

**Comment by agrabeli** [comment]
@hellais Thanks, I've implemented your feedback.

**Comment by hellais** [comment]
This looks good to me, though can you add this to the ooni-web repository?

**Comment by bassosimone** [comment]
@agrabeli I would be happy to merge, is this PR still relevant, tho?

**Comment by hellais** [comment]
This is superseeded by: https://github.com/ooni/probe-android/blob/master/description/source.xlf

**Pull request closed without merging**